src/index.js

```diff
@@ -69,9 +71,13 @@ async function main() {
       core.setFailed(`Assertions have failed.`)
       // continue
     }
+
     core.endGroup() // Asserting
   }
+
   /*******************************UPLOADING************************************/
+  await output.run({ status })
```

Comment:
nit:Is this better named as `notification.run`, or possibly `output.sendNotifications`? "output.run" is a bit ambiguous.